docs/iris/example_code/General/custom_file_loading.py

```diff
@@ -144,7 +145,8 @@ def NAME_to_cube(filenames, callback):
         for i, data_array in enumerate(data_arrays):
             # turn the dictionary of column headers with a list of header information for each field into a dictionary of
             # headers for just this field. Ignore the first 4 columns of grid position (data was located with the data array).
-            field_headings = dict([(k, v[i + 4]) for k, v in column_headings.iteritems()])
+            field_headings = dict((k, v[i + 4])
+                                  for k, v in six.iteritems(column_headings))
```

Comment:
This doesn't happen often enough (i.e. once per cube) to really bother with iteritems() vs items() and muddying the example code with `six`. I suggest switching it to just `... for k, v in column_headings.items()])` @pelson - did you write the original? Are you happy with changing to `items()`?